beeldengeluid/labs.beeldengeluid.nl
# Yarn2npm

this PR replaces all yarn functionality with npm equivalents, which should fix our current issues with dependabot updates not building properly.

Review instructions:

1. pull this branch
2. run `npm install` to install dependencies
3. follow [instructions ](https://github.com/beeldengeluid/labs.beeldengeluid.nl/tree/yarn2npm#development-setup) to start a local development server
4. check that everything works ok
5. sanity check that no rogue yarn references are still hanging around anywhere in the repo
